app/views/calagator/events/_item.html.erb

```diff
@@ -79,21 +79,6 @@ html_classes << " contentbar" if has_contentbar
       <%= format_description(event.description) %>
     </div>
     <% end -%>
-    <div class="attendees">
```

Comment:
Are there styles related to `.attendees` or `.plancast-rsvp` that we should also pull out?

Reply:
Done!